updating 8.7 release note to include CSPM (#3112)
elastic/security-docs: docs/release-notes/8.7.asciidoc

* Improves the UI for building an Investigation Guide query ({pull}150363[#150363]).
* Adds the ability to create a rule from a Timeline ({pull}143020[#143020]).
* Adds the option to suppress custom query rule alerts during a specific time window. Duplicate alerts within that time window will be grouped ({pull}148868[#148868]).

> * Introduces the https://ela.st/cspm[Cloud Security Posture Management (CSPM) feature], which detects misconfigured cloud resources in AWS accounts. 

Reviewer: ```suggestion
* Introduces the <<cspm, Cloud Security Posture Management (CSPM) feature>>, which detects misconfigured cloud resources in AWS accounts. 
```

> * Introduces [Cloud Security Posture Management (CSPM) feature](https://ela.st/cspm) with support for AWS, which continuously detects misconfigured cloud resources in AWS accounts. 

Reviewer: ```suggestion
* Introduces the https://ela.st/cspm[Cloud Security Posture Management (CSPM) feature], which detects misconfigured cloud resources in AWS accounts. 
```
